model_fitting/random_forest.py

```diff
@@ -188,40 +189,37 @@ def train_models(csv_file_path, done_path, logs_dir,error_path, num_of_configura
         os.makedirs(output_path_i, exist_ok=True)
         file_save_configuration = save_configuration_to_txt_file(configuration, output_path_i)
         logging.info(f'Configuration #{i} hyper-parameters are:\n{configuration}')
-        done_file_path=os.path.join(logs_dir, f'{model_number}_done_train_random_forest.txt')
-        cmds=[file_save_configuration, csv_file_path, is_hits_data, output_path_i,
+        done_file_path=os.path.join(logs_dir, f'{model_number}_{csv_file_prefix}_done_train_random_forest.txt')
+        cmds=[file_save_configuration, csv_file_path, rank_method, output_path_i,
                 model_number, done_file_path, '--random_forest_seed', random_forest_seed,
                 '--cv_num_of_splits', cv_num_of_splits]
         if not os.path.exists(done_file_path):
             all_cmds_params.append(cmds)
         else: 
             logger.debug(f'Skipping random forest train as {done_file_path} found')
-           
-    
-    queue_name = 'pupkoweb'
-    verbose = True
-    executable = 'python'
+            num_of_expected_results +=1
+
+      
     if len(all_cmds_params) > 0:
         stop = False
         for count, cmds_params in enumerate(all_cmds_params):
             cmd = submit_pipeline_step(script_name, [cmds_params],
-                                logs_dir, '_done_train_random_forest.txt',
-                                queue_name, verbose, executable=executable)
+                                logs_dir, f'{csv_file_prefix}_{cmds_params[4]}_train_random_forest',

```

Comment:
cmd_params[4] is not clear and to brittle.
If we change to inputs this will break.
Please assign to a variable, e.g. list, when generating the `cmd_params`